features/other_b/bookmark_indexing.feature

```diff
@@ -60,48 +60,59 @@ Feature: Bookmark Indexing
     Then I should not see "Series"
 
   @new-search
-  Scenario: Synning a canonical tag used on a bookmarked series should move the 
-  bookmark to the new canonical's bookmark listings; de-synning should remove it
+  Scenario: Synning a canonical tag used on bookmarked series and external works
+  should move the bookmarks to the new canonical's bookmark listings; de-synning
+  should remove them
     Given I am logged in as "author"
       And a canonical fandom "Veronica Mars"
       And a canonical fandom "Veronica Mars (TV)"
       And I post the work "The Story Telling: Beginnings" with fandom "Veronica Mars" as part of a series "Telling Stories"
       And I am logged in as "bookmarker"
       And I bookmark the series "Telling Stories"
+      And I bookmark the external work "Outside Story" with fandom "Veronica Mars"
     When I go to the bookmarks tagged "Veronica Mars"
-    Then the 1st bookmark result should contain "Telling Stories"
+    Then I should see "Outside Story"
+      And I should see "Telling Stories"
     When I am logged in as a tag wrangler
       And I syn the tag "Veronica Mars" to "Veronica Mars (TV)"
       And I go to the bookmarks tagged "Veronica Mars (TV)"
-    Then the 1st bookmark result should contain "Telling Stories"
+    Then I should see "Outside Story"
+      And I should see "Telling Stories"
     When I de-syn the tag "Veronica Mars" from "Veronica Mars (TV)"
       And the tag "Veronica Mars" is canonized
       And I go to the bookmarks tagged "Veronica Mars (TV)"
     Then I should not see "Telling Stories"
+      And I should not see "Outside Story"
     When I go to the bookmarks tagged "Veronica Mars"
-    Then the 1st bookmark result should contain "Telling Stories"
+    Then I should see "Outside Story"
+      And I should see "Telling Stories"
 
   @new-search
-  Scenario: Subtagging a tag used on a bookmarked series should make the
-  bookmark appear in the metatag's bookmark listings; de-subbing should remove
-  it
+  Scenario: Subtagging a tag used on bookmarked series and external works should
+  make the bookmarks appear in the metatag's bookmark listings; de-subbing
+  should remove them
     Given I am logged in as "author"
-      And a canonical freeform "Alternate Universe"
-      And a canonical freeform "Alternate Universe - High School"
-      And I post the work "The Story Telling: Beginnings" with freeform "Alternate Universe - High School" as part of a series "Telling Stories"
+      And a canonical character "Laura"
+      And a canonical character "Laura Roslin"
+      And I post the work "The Story Telling: Beginnings" with character "Laura Roslin" as part of a series "Telling Stories"
       And I am logged in as "bookmarker"
```

Comment:
This should probably be a `When` as we're switching from posting the work to bookmarking it

Reply:
I'd consider the bookmark creation to be a set up step, since we're not interested in the creation of the bookmark, but the wrangling changes (we don't have "Given a bookmark of the external work/series" steps). So what I'll actually do is get rid of the `When I go to the bookmarks tagged...`bit -- I have no idea why I included that -- and make the switch to `When` when we get to the tag wrangler bit.

Reply:
All sounds good, but I wouldn't worry about it on my account since I only commenting on the `And`/`When` which sounded odd.